submit.go

```diff
@@ -149,7 +149,12 @@ func main() {
 
 	go func() {
 		defer wg.Done()
-		Loop(client, job)
+		job := Loop(client, job)
+		log.Printf("Job status is : %s", job.Status)
+		if job.Status == "COMPLETED" {
+			log.Printf("Job end job loop")
```

Comment:
Shouldn't it be something like this ?
```suggestion
			log.Printf("Job exit code : %v", job.ReturnCode)
```